docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+- An option to change the user agent if needed as well as a button to restart the integrated browser. If Orion gets stuck restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `GitHub <https://github.com/ls1intum/Orion>`_.
+
+Orion for Students
+------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, a button to |open-in-intellij-button| appears.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. Orion provides a button to |submit-button|. This will automatically commit and push the local changes and display the result of the automatic tests in IntelliJ.
+
+`Gif showcasing the usage for students <https://github.com/ls1intum/Orion#example-usage>`_
+
+Orion for Instructors
+---------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the programming exercise creation step 1 and 2.
+ 2. After the creation, navigate to the instructor exercise overview using the integrated browser
+ 3. Each programming exercise will provide a button to |edit-in-intellij-button|. The button is rightmost in the table an might require scrolling. This will download the template, solution and test repository of the exercise
+ 4. Edit the repository files in IntelliJ.
+ 5. Orion provides a button to |submit-button|. This will commit and push all local changes to their respective repository.
+ 6. The integrated browser displays the editor to edit the problem statement.
+ 7. Orion also provides a button to |test-locally-button|, which will copy the local template or solution (whichever was selected) and the tests into a new folder and execute them locally.
+
+Orion for Tutors
+----------------
+
+Tutors can assess programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise's assessment dashboard using the integrated browser
+ 2. There will be a button to |assess-in-orion-button|. It automatically downloads the automatic tests.
+ 3. After downloading or opening the exercise in IntelliJ, the normal submission overview is shown in the integrated browser. Each submission can be opened in Orion, new submissions can be started by |start-assessment-in-orion-button|. This downloads the submission files and overwrites the previous submission.
+ 4. The student's code is in the directory *assignment*. The tests are in the directory *tests*. Additionally, there are two directories *template* and *studentSubmission* that contain internal files and should be ignored.
+ 5. Opening a file from the assignment opens the editor with two available modes, which can be switched using the tabs at the bottom of the editor.
+   - In edit mode ("Text"), the files can be edited regularly, e.g. to try out fixes
```

Comment:
```suggestion
   - In the edit mode ("Text"), the files can be edited regularly, e.g. to try out fixes
```

Reply:
For me this feels wrong, but that might be my lack of english skills.

Reply:
I think the article gets often omitted in common speech/slang but it's actually not correct so in a formal text we should include it.

Reply:
While [this](https://forum.wordreference.com/threads/in-the-mode.2813097/) is not really based on anything, they make a compelling argument that its also "in silent mode", not "in the silent mode".